cli/command/image/list.go

```diff
@@ -75,6 +80,26 @@ func runImages(ctx context.Context, dockerCLI command.Cli, options imagesOptions
 		filters.Add("reference", options.matchName)
 	}
 
+	if options.tree {
+		if options.quiet {
+			return errors.New("--quiet is not (yet) supported with --tree")
+		}
+		if options.noTrunc {
+			return errors.New("--no-trunc is not (yet) supported with --tree")
+		}
+		if options.showDigests {
+			return errors.New("--show-digest is not (yet) supported with --tree")
+		}
+		if options.format != "" {
+			return errors.New("--format is not (yet) supported with --tree")
+		}
```

Comment:
Skip the parentheses?

```suggestion
		if options.quiet {
			return errors.New("--quiet is not yet supported with --tree")
		}
		if options.noTrunc {
			return errors.New("--no-trunc is not yet supported with --tree")
		}
		if options.showDigests {
			return errors.New("--show-digest is not yet supported with --tree")
		}
		if options.format != "" {
			return errors.New("--format is not yet supported with --tree")
		}
```